packages/forms/src/lib/mask/README.md

```diff
@@ -0,0 +1,43 @@
+# @acpaas-ui/ngx-components/forms
+
+With the @acpaas-ui mask module you can mask input fields.
+
+## Usage
+
+```typescript
+import { MaskModule } from '@acpaas-ui/ngx-components/mask'`;
+```
+
+## Documentation
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() auiMask: string;` | - | Use the auiMask input to apply config. |
+
+### Example
+
+```typescript
+import { MaskModule } from '@acpaas-ui/ngx-components/forms';
+
+@NgModule({
+  imports: [
+    MaskModule,
+  ]
+});
+
+export class AppModule {};
+```
+
+```html
+<div class="a-input">
+	<input type="text" placeholder="dd/mm/yyyy" auiMask="99/99/9999" />
```

Comment:
This is a very basic example.
Perhaps add a bit more based on the original documentation or on the docs of the `Inputmask` package?

Reply:
Maybe it's better to come up with some examples that developers will actually need, like a "bank account number" mask or a" national insurance number" mask.